chapters/lexicalstructure.tex

```diff
@@ -251,7 +251,7 @@ \subsection{Units of Literal Constants}\label{units-literal-constants}
 
 The following rules apply:
 - The result of `x + L`, `L + x`, `x - L`, `L - x`, `x*L`, `L*x`, `x/L`, where `x` is an expression with non-empty \lstinline!unit! attribute string `<s>` and `L` is an expression containing only literal constants, shall have the same \lstinline!unit! attribute string of `x`.
-- The result of `L/x` shall the the \lstinline!unit! attribute string `1/(<s>)`. 
+- The result of `L/x` shall the the \lstinline!unit! attribute string `1/(<s>)`.
```

Comment:
The other part of that fix  (from [here](https://github.com/modelica/ModelicaSpecification/pull/3266#discussion_r1018728122)) was omitted - "shall **have** the" - could you also commit that, Hans? :-)

Reply:
Ok reluctantly done.
In general I don't want to commit so much directly, but I made an exception since it didn't build so we couldn't look at the result.

Reply:
Thanks!

Reply:
This conversation can be resolved (I don't have that button).